test/parallel/test-http2-server-shutdown-redundant.js

```diff
@@ -9,6 +9,12 @@ const http2 = require('http2');
 const server = http2.createServer();
 
 server.on('stream', common.mustCall((stream) => {
+  stream.once('error', common.expectsError({
+    name: 'Error',
+    code: 'ERR_HTTP2_STREAM_ERROR',
+    message: 'Stream closed with error code NGHTTP2_INTERNAL_ERROR'
+  }));
```

Comment:
Need to check if this emit is correct.

Reply:
It comes from `nghttp2` and is a result of closing the entire session. `session.destroy()` sends RST to each stream first. Also see https://github.com/nodejs/node/pull/53160#discussion_r1619123266